### DISCUSSION TITLE
Ability to set root view background color to prevent react-navigation white flash

### DISCUSSION THREAD

**Discussion by efstathiosntonas** [jpudysz/react-native-unistyles#190] [Feature Requests]
@jpudysz creating a separate discussion for this to keep track of it if you decide to implement.

Ability to set root view background color like this lib is doing it: https://github.com/pnthach95/react-native-root-view-background

The goal of the lib above is to prevent react-navigation weird white flash on screen navigate and rotation of the screen under certain circumstances by setting the background color outside of the React tree.

Expo got this already as a package: https://docs.expo.dev/versions/latest/sdk/system-ui/ so for the bare react-native devs it's nice to have

**Comment by efstathiosntonas** [comment]
this could be expanded to web too I believe (expo got it so why not)

**Comment by jpudysz** [comment]
Implemented in the latest RC
